bower.json

```diff
@@ -34,12 +34,12 @@
     "angular-resource": "1.5.11",
     "angular-sanitize": "1.5.11",
     "angular-route": "1.5.11",
-    "angular-bootstrap": "0.14.3",
-    "angular-patternfly": "3.24.0",
+    "angular-bootstrap": "2.2.x",
```

Comment:
You're right: we can. But I'd prefer to keep them at the same level. If we (unintentionally) use a new angular-bootstrap API here, it will work, but break when we pull it into console. Those kinds of errors are hard to track down.

I don't think the changes are too hard to just bump it everywhere. See my comments in https://github.com/openshift/origin-web-console/issues/1861